package.json

```diff
@@ -39,9 +40,10 @@
     "geofirestore": "^3.4.1",
     "googleapis": "^48.0.0",
     "leaflet": "^1.6.0",
-    "nl2br-pipe": "^1.2.3",
+    "ngx-cookieconsent": "^2.2.3",
     "ngx-lightbox": "^2.1.2",
     "ngx-star-rating": "0.0.1",
+    "nl2br-pipe": "^1.2.3",
```

Comment:
Do we need it for the consent bar or why it’s included?

Reply:
It just moved a view lines further down. It is needed - and already included for a couple of weeks - to format crlf in user text to be displayed properly on trader-detail page f.e.